InternLM/HuixiangDou
# WIP: feat(web/proxy): update

* simplify `llm_client.py`
  * add pytest for auto fix llm backend
  * add log error in `llm_client.py` for LLM API return error
  * usage `pytest ./unittest/`
* simplify `llm_hybrid_server.py`
  * LLMs use the same error handling and exponential backoff strategy
  * add relative pytest
* support `work.time.enable` in `config.ini` 
  * If false, skip work time check
* add RAG tool for SFT data
* fix throttle for web version
* typo

![image](https://github.com/InternLM/HuixiangDou/assets/7872421/7171f987-a680-47ac-841a-271bc6503cd2)
